docs/index.md

```diff
@@ -14,10 +14,24 @@ See the [LFPH Landscape](https://landscape.lfph.io/category=gaen-bluetooth&forma
-* **PHA** Public Health Authority
-* **LFPH** Linux Foundation Public Health
+It is imperative that we are precise as we communicate about exposure notification technologies.
+Even slight mischaracterizations can undermine efforts to build trust amongst the public.
+Here, we aggregate some key terms, their meanings, and the context in which they are typically used.
+
+| **Term** | **Definition** | **Context** |
+|------|------------|---------|
+| **Anonymous identifiers** | Unique values that do not carry personally identifying information nor can they be used to identify an individual | In exposure notification applications, anonymous identifiers are broadcast to devices whose proximity has led to a contact event. |
+| **Contact event** | When two users have been within certain proximity for a specified amount to time | The proximity distance and specified amount of time are typically defined by the relevant public health agency. An example of a contact event could be when two people are within 6 feet of each other for 15 minutes. Within EN apps, events that do not meet the criteria are not recorded. |
+| **Contact tracing** | A (typically manual) process for recording and notifying people who an infected person has come in to contact with | This term should be used to refer to the manual process, not to exposure notification applications as these applications do not track or trace users and their contacts |
+| **GAEN** | Google/Apple Exposure Notification | You will often see this referring to the GAEN API (Application Programming Interface) or the GAEN Framework, technologies provided by Google and Apple to be used by development teams to build apps with the ability to perform private proximity detection and alerting. |
+| **Implementer** | The team that is responsible for the technical execution of the exposure notification app | Usually, but not always, this is a technology firm hired by the PHA. |
+| **PHA** | Public Health Authority | PHAs are typically associated with the government of a certain locality. |
+| **Proximity** | A measure of distance between items, not associated with location | EN app measure the proximity between smartphones to understand whether a contact event has occurred. This allows notification of potential exposure without having to collect user locations. |
+| **LFPH** | Linux Foundation Public Health | As an open source community, Linux Foundation's new Public Health initiative combines efforts of multiple groups working toward privacy-preserving technologies, initially geared toward technology to aid in the fight against COVID-19 |
+| **Surveillance** | Monitoring or observing activity | Epidemiologists sometimes use this term to refer to their work to understand the spread and containment of a virus. Even though EN apps can aid in containment efforts, privacy-preserving apps do not perform surveillance from a technology perspective as they do not collect location, track, or record user activity. |
```

Comment:
There's a lack of parallelism in this final list - I think it should be "they do not collect, track, or record user location or history." ?